nodejs-assets/nodejs-project/sample-main.js

```diff
@@ -1,12 +1,12 @@
 // Rename this sample file to main.js to use on your project.
 // The main.js file will be overwritten in updates/reinstalls.
 
-var rn_bridge = require('rn-bridge');
+var rnBridge = require('rn-bridge')
 
 // Echo every message received from react-native.
-rn_bridge.channel.on('message', (msg) => {
-  rn_bridge.channel.send(msg);
-} );
+rnBridge.channel.on('message', (msg) => {
+  rnBridge.channel.send(msg)
+})
 
 // Inform react-native node is initialized.
-rn_bridge.channel.send("Node was initialized.");
\ No newline at end of file
+rnBridge.channel.send('Node was initialized.')

```

Comment:
Could you make the linter ignore this file? It's not supposed to be linted because if/when nodejs-mobile updates, it will run a script to edit this file, and the diff would tell us how to properly update the backend. Note that this file is `sample-main.js`

Reply:
done